src/containers/Routes/components/Drawer/Drawer.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { Fragment } from "react";
 import { TouchableOpacity } from "react-native";
-import { WrapperContainer, Typography, Divider } from "../../../../components";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { Typography, Divider } from "../../../../components";
 import { drawerContent } from "./Drawer.helpers";
 import { styles } from "./Drawer.styles";
 
 export const DrawerMenu = () => {
   return (
-    <WrapperContainer>
+    <SafeAreaView style={[styles.mainContainer]}>
```

Comment:
You can once wrap your app into SafeAreaView instead of using it in each component

Reply:
It was used in header and drawer, now that drawer use DrawerContent, its only header